staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/plugin.go

```diff
@@ -92,5 +92,11 @@ func (a *Plugin) ValidateInitialization() error {
 
 // Admit makes an admission decision based on the request attributes.
 func (a *Plugin) Admit(attr admission.Attributes) error {
-	return a.Webhook.Dispatch(attr)
+	if !a.WaitForReady() {
```

Comment:
Why moving this check from `Dispatch` to here? The old ways saves some duplicate code.